chef-utils/lib/chef-utils/dsl/cloud.rb

```diff
@@ -30,7 +30,8 @@ module Cloud
       # @return [Boolean]
       #
       def cloud?(node = __getnode)
-        node.key?("cloud")
+        # cloud is always present, but nil if not on a cloud
+        !node["cloud"].nil?
```

Comment:
```suggestion
        !node["cloud"].nil?
```

I feel like `!!node["cloud"]` would be better in case whatever is setting this value ever sets it to an explicit `false`.

Reply:
Oh oops, I read further and saw in the tests that is a hash, not a boolean. In that case, maybe a check for `.empty?` is called for?

Reply:
On a non-cloud system the value is nil though. The tests are giving the empty hashes to say "there would be some data here, but I'm not sticking 100 lines of cloud data into the spec"

Reply:
Gotcha 👍